src/globals/scss/styles.scss

```diff
@@ -6,6 +6,7 @@ $css--font-face: false !default;
 $css--helpers: true !default;
 $css--body: true !default;
 $css--reset: false !default;
+$css--use-layer: true !default;
```

Comment:
IMO, It's the naming that needs to change. 

For example, $css--body outputs CSS with a selector and styles:
```css
  h1 {
    @include reset;
    @include font-size('32');
    @include line-height('heading');
    @include letter-spacing;
 }
```

But `$css--use-layer` is only going to output styles and no selector:

```css
@mixin layer($layer) {
  @if global-variable-exists('css--use-layer') == false or $css--use-layer == true  {
    @if map-has-key($layer-shadows, $layer) {
      box-shadow: #{map-get($layer-shadows, $layer)};
    }
    @else {
      @warn '#{$layer} is not a valid layer.';
    }
  }
}
```